netutils/codecs/md5.c

```diff
@@ -119,15 +119,15 @@ static void byteReverse(FAR unsigned char *buf, unsigned longs)
  ****************************************************************************/
 
 /****************************************************************************
- * Name: MD5Init
+ * Name: md5_init
  *
  * Description:
  *   Start MD5 accumulation.  Set bit count to 0 and buffer to mysterious
  *   initialization constants.
  *
  ****************************************************************************/
 
-void MD5Init(struct MD5Context *ctx)
+void md5_init(struct md5_context_s *ctx)
```

Comment:
This is a global function.  This will need a corresponding change to the prototype in apps/include/netutils/md5.h